tests/stub/routing/test_routing_v4x3.py

```diff
@@ -426,18 +429,8 @@ def work(tx):
         session.close()
         driver.close()
 
-        self._routingServer1.done()
-        self._writeServer1.done()
-        if self.driver_supports_features(types.Feature.OPT_CONNECTION_REUSE):
-            self.assertEqual(self._writeServer1.count_responses("<ACCEPT>"), 1)
-        else:
-            self.assertLessEqual(
-                self._writeServer1.count_responses("<ACCEPT>"), 2
-            )
-        self.assertEqual([[1], [1]], sequences)
-        self.assertEqual(self.route_call_count(self._routingServer1), 2)
```

Comment:
why did all of this go?

Reply:
Sorry, this was accidentally removed during deletion of a new test that should come in a future PR.
I have restored it now.